certif/tests/unit/components/enrolled-candidates_test.js

```diff
@@ -70,6 +70,53 @@ module('Unit | Component | enrolled-candidates', function(hooks) {
       assert.equal(component.args.certificationCandidates.length, 1);
     });
   });
+  module('#addCertificationCandidateInStaging', async function() {
+
+    module('When FT_IS_NEW_CPF_DATA_ENABLED is enabled', async function() {
+
+      test('should not add an empty candidate in staging', async function(assert) {
+      // given
+        const sessionId = 'sessionId';
+        const certificationCandidateData = _buildCandidate({});
+        const savableCandidate = _buildCandidate(
+          { ...certificationCandidateData },
+          { save: sinon.stub().resolves(), deleteRecord: sinon.stub() });
+        const store = { createRecord: sinon.stub().returns(savableCandidate) };
+
+        component.store = store;
+        component.args = {
+          isNewCpfDataToggleEnabled: true,
+          certificationCandidates: [_buildCandidate({
+            ...certificationCandidateData,
+          })],
+          sessionId,
+        };
+
+        // when
+        await component.addCertificationCandidateInStaging();
+
+        // then
+        assert.equal(component.candidatesInStaging.length, 0);
+      });
+
+    });
+    module('When FT_IS_NEW_CPF_DATA_ENABLED is  not enabled', async function() {
+
+      test('should add an empty candidate in staging', async function(assert) {
```

Comment:
Peut-être qu'en modifiant le texte du test ce serait moins confus, rendre clair qu'on a une volonté de rendre inactif le bouton.
should disable the feature of enrolling a new candidate
ou un truc du style

Reply:
J'y ai pensé mais j'avais peur que le 'disable' ne mette sur la fausse piste d'un bouton avec l'attribut disabled